src/Umbraco.AuthorizedServices/Manifests/AuthorizedServicesManifestFilter.cs

```diff
@@ -10,7 +10,7 @@ public void Filter(List<PackageManifest> manifests)
         {
             AllowPackageTelemetry = true,
             Version = Constants.InformationalVersion,
```

Comment:
Umbraco 12 supports automatically retrieving the version from an assembly when either the `PackageId` or `VersionAssemblyName` are specified in the manifest, but will also already set the version when a package migration with the same ID/name exists. So this can be removed if a v12+ only version is released...